### ISSUE TITLE
Reach out to other direnv features

### ISSUE THREAD

**Issue by jcbhmr** [devcontainers-community/features-direnv#1]
Hello @shikanime! 👋

I noticed you opened https://github.com/devcontainers-contrib/features/pull/501 and I figured you might be interested in this project.

**Comment by shikanime** [comment]
@jcbhmr Yes definitly! What's the relation between devcontainers-community<->devcontainers-contrib<->devcontainers ?

**Comment by jcbhmr** [comment]
The gist of it at present is:
- @devcontainers: the official Microsoft organization devoted to standardizing and managing official stuffs. they do the spec, a small set of official features + templates and some base images.
- @devcontainers-contrib: @danielbraun89's awesome collection of devcontainer features. currently limited by single-human bandwidth of the owner.
- @devcontainers-community: intended to be run by _many_ and cover anything not quite official-level popular

here's a discussion thingie since someone else had the same question in https://github.com/devcontainers-community/features-bazel/issues/1 https://github.com/orgs/devcontainers-community/discussions/7
